inst/json/graph_prevalence_config.json

```diff
@@ -1,24 +1,28 @@
 {
     "metadata": {
-          "x_col": "month",
-          "y_col": "value",
-          "id_col": "intervention",
-          "format": "long"
+        "x_col": "month",
+        "y_col": "value",
+        "id_col": "intervention",
+        "format": "long"
     },
     "layout": {
-           "title": "Projected prevalence in under 5 year olds",
-           "xaxis": {
-               "title": "years of intervention",
-               "showline": true,
-               "tickvals": [12, 24, 36],
-               "ticktext": [1, 2, 3]
-           },
-           "yaxis": {
-               "title": "prevalence (%)",
-               "showline": true,
-               "range": [0, 1],
-               "autorange": false
-           }
+        "title": "Projected prevalence in under 5 year olds",
+        "xaxis": {
+            "title": "years of intervention",
+            "showline": true,
+            "tickvals": [12, 24, 36],
+            "ticktext": [1, 2, 3]
+        },
+        "yaxis": {
+            "title": "prevalence (%)",
+            "showline": true,
+            "autorange": true,
+            "rangemode": "tozero",
+            "tickformat": ".1%"
```

Comment:
We get absolute prevalence values from the data, but want to show these as % in the plot axis...